.github/dependabot.yml

```diff
@@ -1,162 +1,26 @@
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "config/derive"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "core"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "core/docs"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "core/test_network"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "crypto"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "crypto_cli"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "data_model"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "dsl"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "futures"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "futures/derive"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
+  - package-ecosystem: "github-actions"
     target-branch: "iroha2-dev"
-    directory: "logger"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "macro"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "macro/derive"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "p2p"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "permissions_validators"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "schema"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "schema/bin"
-  - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "schema/derive"
-  - package-ecosystem: "cargo"
+    directory: "/"
     schedule:
-      interval: "daily"
+      interval: "weekly"
+
+  - package-ecosystem: "pip"
     target-branch: "iroha2-dev"
-    directory: "substrate"
-  - package-ecosystem: "cargo"
+    directory: "/client_cli/pytests/"
     schedule:
       interval: "daily"
-    target-branch: "iroha2-dev"
-    directory: "telemetry"
+
   - package-ecosystem: "cargo"
-    schedule:
-      interval: "daily"
     target-branch: "iroha2-dev"
-    directory: "version"
-  - package-ecosystem: "cargo"
+    directory: "/"
```

Comment:
Would this work for non-workspace crates, like `iroha_default_validator`?

Reply:
good point, I think not